CONTRIBUTING.md

```diff
@@ -4,39 +4,50 @@
 The following set of guidelines will help you in contributing to Arcus repositories.
 But, no worries, if you forget something or have problems with some of these steps, we're happy to help! In those cases: create a PullRequest on the repository so we can talk about it.
 
-This contribution guide has the following contents:
+This contribution guide has the following content:
+- [Issues and discussions](#issues-and-discussions)
+- [Pull Requests](#pull-requests)
+  - [Forking](#forking)
+  - [Implementing changes](#implementing-changes)
+  - [Review and merge of the changes](#review-and-merge-of-the-changes)
+- [Project structure](#project-structure)
+- [Way of working](#way-of-working)
+- [Code guidelines](#code-guidelines)
+  - [General](#general)
+  - [Testing](#testing)
+- [Documentation](#documentation)
```

Comment:
I’m normally not a fan of MD table of contents, as they quickly get out-of-date; especially in open-source, where people use diff IDE tools all the time and these things don’t get updated automatically.

I’d be hesitant to include it here; but if you feel like this is a really big plus, we can leave it in.

Reply:
I think it offers a quick way to jump to certain sections.  I'd leave it in :)

Reply:
It indeed is easy to jump to certain sections for people who do use it. It getting out of date doesn't seem like a realistic risk to me as all changes are reviewed, so it would be spotted if it's not updated.
I would leave it in as well as it would be beneficial for the people who use it :)

Reply:
We can add that GitHub action as a PR status check that notifies broken links, that could help, too.